### DISCUSSION TITLE
Node not recognized as an internal or external command in Browsershot

### DISCUSSION THREAD

**Discussion by UserAhmad2001** [spatie/browsershot#770] [Q&A]
I am getting the error message "node is not recognized as an internal or external command, operable program or batch file" when I try to run the following command:

node "C:\Users\kinga\OneDrive\Documents\webpages\Chatbott\vendor\spatie\browsershot\src/../bin/browser.cjs" "{""url"":""https://www.amazon.com"",""action"":""screenshot"",""options"":{""type"":""png"",""path"":""assets/img/screenshot.jpg"",""args"":[],""viewport"":{""width"":800,""height"":600}}}""

Below is screenshot route code:
`
Route::get('/screenshot', function () {
    Browsershot::url('https://amazon.com')->save('assets/img/screenshot.jpg');
    return redirect('/');
});
`

I have tried adding the path to the node executable to the system path, but the issue still persists. I have also tried restarting my computer and running the command as an administrator, but the issue still persists.

I am using the following system:

Operating system: Windows 10
Node version: 16.13.0
Browsershot version: 4.2.0
I have attached the full error message to this discussion.

I would appreciate it if someone could look into this issue and let me know if there is a workaround or if a fix is in the works.

Thank you for your time and attention to this matter.

**Comment by cas-technology** [comment]
Make sure you installed [node](https://nodejs.org/en/download) correctly., verify you installed it correctly by running `node --version` in the same window you ran the above command.

**Comment by UserAhmad2001** [comment]
I have already tried that, node is installed
